lib/sierra_db_client.py

```diff
@@ -0,0 +1,60 @@
+import os
+from nypl_py_utils import (PostgreSQLClient)
+from psycopg.rows import dict_row
+from nypl_py_utils.functions.log_helper import create_log
+
+
+class SierraDbClient:
+    def __init__(self):
+        self.base_client = PostgreSQLClient(
+            os.environ['SIERRA_DB_HOST'],
+            os.environ['SIERRA_DB_PORT'],
+            os.environ['SIERRA_DB_NAME'],
+            os.environ['SIERRA_DB_USER'],
+            os.environ['SIERRA_DB_PASSWORD']
+        )
+        self.logger = create_log('sierra_db_client')
+
+    def holdshelf_entries(self):
+        self.base_client.connect()
+
+        # Cast rows to dicts:
+        with self.base_client.pool.connection() as conn:
```

Comment:
This seems like something that should be added to the main utils instead (which I'm happy to take on)